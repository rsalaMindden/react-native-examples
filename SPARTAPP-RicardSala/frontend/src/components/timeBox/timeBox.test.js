import React from "react";
import {render} from '../../utils/tetsUtils/test.utils'

import Timer from "./timeBox";


const time={
    time:jest.fn()
}

describe('Given a Timer component',()=>{
    describe('When it is render',()=>{
        let screen
        beforeEach(()=>{
            screen=render(<Timer  hoursMinSecs={time}/>)
        })
        test('Then should match the snapshot',()=>{
            expect(screen).toMatchSnapshot();
          })
        })
    })