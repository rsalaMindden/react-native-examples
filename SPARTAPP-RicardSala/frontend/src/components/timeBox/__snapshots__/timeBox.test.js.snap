// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Given a Timer component When it is render Then should match the snapshot 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
      },
    ]
  }
>
  <View
    accessible={true}
    collapsable={false}
    focusable={true}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
  >
    <Text
      style={
        Array [
          Object {
            "color": "#FFD523",
          },
          Object {
            "fontSize": 100,
            "fontWeight": "600",
          },
        ]
      }
    >
      00:60
    </Text>
  </View>
</View>
`;
